crates/walrus-sdk/src/client/communication/node.rs

```diff
@@ -91,31 +91,32 @@ impl<T, E> WeightedResult for NodeResult<T, E> {
     }
 }
 
-pub(crate) struct NodeCommunication<'a, W = ()> {
+#[derive(Debug)]
+pub struct NodeCommunication<W = ()> {
     pub node_index: NodeIndex,
     pub committee_epoch: Epoch,
-    pub node: &'a StorageNode,
-    pub encoding_config: &'a EncodingConfig,
+    pub node: Arc<StorageNode>,
```

Comment:
Question: Is there any benefit of using an `Arc` here over just an owned `StorageNode`?